pac4j-oauth/src/main/java/org/pac4j/oauth/client/GenericOAuth20Client.java

```diff
@@ -56,6 +73,49 @@ protected void clientInit() {
         super.clientInit();
     }
 
+    private Class[] getConverters() {
+        if (converterClasses == null) {
+            try {
+                ClassLoader classLoader = Thread.currentThread().getContextClassLoader();
+                Class cla = classLoader.getClass();
+                while (cla != ClassLoader.class) {
+                    cla = cla.getSuperclass();
+                }
+                Field field = cla.getDeclaredField("classes");
+                field.setAccessible(true);
+                Vector<Class> classes = (Vector<Class>) field.get(classLoader);
```

Comment:
I don't like the idea of browsing all classes to find the right we want: what about the performance?

Cannot we just initialize a `Map`  with the existing converters instead of using reflection?

Reply:
There is a cached object "converterClasses" in the GenericOAuth20Client instance that reflects only once,
If we use `Map`, that would be inflexible, and in the future, when we added the converter, we also need to add it here, and maybe we will miss it.

Reply:
Yes, I understand. Can you update the documentation then? -> https://github.com/pac4j/pac4j/blob/master/documentation/docs/clients/oauth.md

Reply:
OK, I'll do it